Sherlock/yoda/DOCU.md

```diff
@@ -355,11 +369,14 @@ api.Transaction([](PrimesAPI::T_DATA data) {
 // Primes that start with `4`.
 EXPECT_EQ("41,43,47",
           api.Transaction([](PrimesAPI::T_DATA data) {
-            // TODO(dkorolev): In real life, the order can be off. -- D.K.
             std::ostringstream os;
+            std::set<int> n;
             for (const auto cit : data[static_cast<FIRST_DIGIT>(4)]) {
-              os << ',' << (static_cast<int>(cit.row) * 10 +
-                            static_cast<int>(cit.col));
+              n.insert(static_cast<int>(cit.row) * 10 +
+                       static_cast<int>(cit.col));
+            }
+            for (int i : n) {
```

Comment:
Ah, right. LGTM this change then. Are you doing anything else with it, or
good to merge? :)

On Thursday, June 11, 2015, Maxim Zhurovich <notifications@github.com>
wrote:

> In Sherlock/yoda/DOCU.md
> <https://github.com/c5t/Current/pull/204#discussion_r32257378>:
>
> >              for (const auto cit : data[static_cast<FIRST_DIGIT>(4)]) {
> > -              os << ',' << (static_cast<int>(cit.row) * 10 +
> > -                            static_cast<int>(cit.col));
> > +              n.insert(static_cast<int>(cit.row) * 10 +
> > +                       static_cast<int>(cit.col));
> > +            }
> > +            for (int i : n) {
>
> Ok, it can accept std::set<std::string>, but here I have std::set<int>
> and change will require additional to_string.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/c5t/Current/pull/204/files#r32257378>.
>


-- 
And, as always, have a nice day!
Dima

http://dimakorolev.com/ | http://www.linkedin.com/in/dimakorolev |
dmitry.korolev@gmail.com | +1 (312) 593 2783